packages/core/src/variables/VariableProcessor.ts

```diff
@@ -182,18 +182,16 @@ class VariableProcessor {
     if (!(includeElement.children && includeElement.children.length)) {
       return {};
     }
-    const includeChildVariables: { [key: string]: string } = {};
-    const includeElementAttribs = includeElement.attribs as { [s: string]: string };
+    const includeChildVariables: Record<string, string> = {};
 
     includeElement.children.forEach((child) => {
       if (child.name !== 'variable' && child.name !== 'span') {
         return;
       }
-      const childAttribs = child.attribs as { [s: string]: string };
-      const variableName = childAttribs.name || childAttribs.id;
+      const variableName = child.attribs && (child.attribs.name || child.attribs.id);
```

Comment:
> if `child.attribs` exists, then `variableName` will be `child.attribs.name` (if it exists) or `child.attribs.id` (otherwise)?

Yup! `child` is of type DomElement so we can't assert that it has `name` and `attribs` attributes. If `child.attribs` doesn't exist then it will be caught in the `variableName` check in the next line.

Reply:
Can we improve the readability here? Is variableName a bool?

Reply:
Ah just realised a better way for this: `const variableName = child.attribs?.name || child.attribs?.id;`
Thanks for highlighting!